docs/api/tray.md

```diff
@@ -60,9 +60,10 @@ rely on the `click` event and always attach a context menu to the tray icon.
 
 ### `new Tray(image)`
 
-* `image` ([NativeImage](native-image.md) | String)
+* `image` ([NativeImage](native-image.md) | String | null)
```

Comment:
Although less semantic I'm not a fan of having an argument that can be `null` on one platform but would throw errors on another.  A static type checker (typescript for example) wouldn't be able to warn you about these issues.

Can we instead recommend that uses provide `nativeImage.createEmpty()` if they wish to have a tray with no icon?

Reply:
@MarshallOfSound I will investigate that